render/detailed/censor.go

```diff
@@ -0,0 +1,48 @@
+package detailed
+
+import (
+	"github.com/weaveworks/scope/report"
+)
+
+func censorNodeSummary(s NodeSummary, cfg report.CensorConfig) NodeSummary {
+	if cfg.HideCommandLineArguments && s.Metadata != nil {
+		// Iterate through all the metadata rows and strip the
+		// arguments from all the values containing a command
+		// (while making sure everything is done in a non-mutable way).
+		metadata := []report.MetadataRow{}
+		for _, row := range s.Metadata {
+			if report.IsCommandEntry(row.ID) {
+				row.Value = report.StripCommandArgs(row.Value)
```

Comment:
It shouldn't since:

* `range` in the loop above returns a copy of `row` and we're changing that copy.
* The test is passing and normally it should fail if any of these operations are mutating since we're not resetting the test set between test cases for this test.